docker/client.Dockerfile

```diff
@@ -1,11 +1,12 @@
 FROM node
 
 COPY web/package.json /client/package.json
+COPY web/yarn.lock /client/yarn.lock
```

Comment:
I know this isn't what the PR is addressing, but I still don't see why the build is being done this way. @satra said it was to prevent running yarn install repeatedly, however when I run `docker build` locally, unless the actual dependencies change (which requires a yarn install anyway), docker caches that step.

Reply:
Not sure if this differs from your experience, but here's what should happen:

1) when lockfile or package.json change, a full docker build happens.
2) when only code changes, docker should cache the yarn install

You're saying this was the behavior even before @satra's change?